Cargo.toml

```diff
@@ -25,21 +25,13 @@ members = [
 # need this because of bifrost farming dependency in runtime
 # bifrost farming uses different orml-traits for orml-currencies
 [patch."https://github.com/open-web3-stack/open-runtime-module-library"]
-orml-traits = { git = "https://github.com/open-web3-stack//open-runtime-module-library", branch = "polkadot-v0.9.42" }
-
-# need this because of bifrost farming dependency in runtime
-# bifrost uses :
-# orml packages { version = "0.4.1-dev" }
-# zenlink packages  { version = "*" }
-[patch.crates-io]
-orml-traits = { git = "https://github.com/open-web3-stack//open-runtime-module-library", branch = "polkadot-v0.9.42" }
-orml-currencies = { git = "https://github.com/open-web3-stack//open-runtime-module-library", branch = "polkadot-v0.9.42" }
-orml-tokens = { git = "https://github.com/open-web3-stack//open-runtime-module-library", branch = "polkadot-v0.9.42" }
-zenlink-protocol = { git = "https://github.com/zenlinkpro/Zenlink-DEX-Module", branch = "polkadot-v0.9.42" }
-zenlink-protocol-runtime-api = { git = "https://github.com/zenlinkpro/Zenlink-DEX-Module", branch = "polkadot-v0.9.42" }
-
-
-[patch.'https://github.com/paritytech/substrate']
-pallet-vesting = { git = "https://github.com/pendulum-chain/patch-pallet-vesting-transaction-storage", branch = "polkadot-v0.9.42" }
-pallet-transaction-payment = { git = "https://github.com/pendulum-chain/patch-pallet-vesting-transaction-storage", branch = "polkadot-v0.9.42" }
+orml-traits = { git = "https://github.com/open-web3-stack//open-runtime-module-library", branch = "polkadot-v1.1.0" }
+orml-tokens = { git = "https://github.com/open-web3-stack//open-runtime-module-library", branch = "polkadot-v1.1.0" }
+##
+### need this because of bifrost farming dependency in runtime
+### bifrost uses :
+### orml packages { version = "0.4.1-dev" }
+### zenlink packages  { version = "*" }
 
+[patch."https://github.com/w3f/ring-vrf"]
+bandersnatch_vrfs = { git = "https://github.com/w3f//ring-vrf?rev=3ddc20", version = "0.0.4", rev = "3ddc20" }
```

Comment:
So... I could run: 
```
cargo b -p amplitude-runtime --features runtime-benchmarks
``` 
in `main` branch with no problems.

But now:
```
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
note: for more details see https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions
    Updating git repository `https://github.com/w3f//ring-vrf?rev=3ddc20`
fatal: https://github.com/w3f//ring-vrf?rev=3ddc20/info/refs not valid: is this a git repository?
error: failed to load source for dependency `bandersnatch_vrfs`

Caused by:
  Unable to update https://github.com/w3f//ring-vrf?rev=3ddc20?rev=3ddc20

Caused by:
  failed to fetch into: /Users/b.carlayap/.cargo/git/db/ring-vrf-5f0e10e9eb5881f6
```

If you encountered this, do you know the solution?

Reply:
```
cargo b -p amplitude-runtime --features runtime-benchmarks
```
works for me on `polkadot-v1.1.0`. Did you `cargo clean` before building on the other branch?

Reply:
Yes, please run `cargo clean` or `cargo update -p bandersnatch_vrfs`, since probably the patch needs to be applied on your local setup @b-yap.

Reply:
@gianfra-t  
I forgot to mention: `cargo clean` and `cargo update` didn't work for me because of: 
**`https://github.com/w3f//ring-vrf?rev=3ddc20?rev=3ddc20`**

`rev` part is called twice; probably because it is specified in the `git` field, followed by `rev` field.

I updated to 
```
[patch."https://github.com/w3f/ring-vrf"]
bandersnatch_vrfs = { git = "https://github.com/w3f//ring-vrf", version = "0.0.4", rev = "3ddc20" }
```
and the error disappeared.

Reply:
Thanks for finding this! I will change to the updated one.